src/main/resources/application.properties

```diff
@@ -41,17 +41,31 @@ endpoint.mongodb.company_profile.cache.key=mongoCompanyProfile
 endpoint.mongodb.disqualifications.cache.key=mongoDisqualifications
 
 # Timers
-endpoint.company_count.timer=timer:company_count?repeatCount=1&delay=${COMPANY_COUNT_DELAY}
-endpoint.company_collection.timer=timer:company_collection?repeatCount=1&delay=${COMPANY_NUMBER_MONGO_ORACLE_DELAY}
-endpoint.company_collection_mongo_primary.timer=timer:company_collection_mongo_primary?repeatCount=1&delay=${COMPANY_NUMBER_MONGO_PRIMARY_DELAY}
-endpoint.company_collection_mongo_alpha.timer=timer:company_collection_mongo_alpha?repeatCount=1&delay=${COMPANY_NUMBER_MONGO_ALPHA_DELAY}
-endpoint.dsq_officer_collection.timer=timer:dsq_officer_collection?repeatCount=1&delay=${DISQUALIFICATION_DELAY}
+endpoint.company_count_mongo_oracle.timer=timer:company_count_mongo_oracle?repeatCount=1&delay=${COMPANY_COUNT_MONGO_ORACLE_DELAY}
+endpoint.company_number_mongo_oracle.timer=timer:company_number_mongo_oracle?repeatCount=1&delay=${COMPANY_NUMBER_MONGO_ORACLE_DELAY}
+endpoint.company_number_mongo_primary.timer=timer:company_number_mongo_primary?repeatCount=1&delay=${COMPANY_NUMBER_MONGO_PRIMARY_DELAY}
+endpoint.company_number_mongo_alpha.timer=timer:company_number_mongo_alpha?repeatCount=1&delay=${COMPANY_NUMBER_MONGO_ALPHA_DELAY}
+endpoint.dsq_officer_id_mongo_oracle.timer=timer:dsq_officer_id_mongo_oracle?repeatCount=1&delay=${DSQ_OFFICER_ID_MONGO_ORACLE_DELAY}
 endpoint.company_name_mongo_primary.timer=timer:company_name_mongo_primary?repeatCount=1&delay=${COMPANY_NAME_MONGO_PRIMARY_DELAY}
 endpoint.company_name_mongo_alpha.timer=timer:company_name_mongo_alpha?repeatCount=1&delay=${COMPANY_NAME_MONGO_ALPHA_DELAY}
 endpoint.company_status_mongo_primary.timer=timer:company_status_mongo_primary?repeatCount=1&delay=${COMPANY_STATUS_MONGO_PRIMARY_DELAY}
 endpoint.company_status_mongo_alpha.timer=timer:company_status_mongo_alpha?repeatCount=1&delay=${COMPANY_STATUS_MONGO_ALPHA_DELAY}
 endpoint.company_status_mongo_oracle.timer=timer:company_status_mongo_oracle?repeatCount=1&delay=${COMPANY_STATUS_MONGO_ORACLE_DELAY}
 
+# Trigger Route Toggles
+company_count_mongo_oracle_enabled=${COMPANY_COUNT_MONGO_ORACLE_ENABLED:true}
```

Comment:
Let's remove the default value on the right-hand side so that these environment variables are mandatory

Reply:
Addressed in latest commit.